beater/connbeat.go

```diff
@@ -76,6 +77,9 @@ func toMap(containerInfo *ContainerInfo) common.MapStr {
 			"id":        containerInfo.id,
 			"local_ips": containerInfo.localIPs.ToSlice(),
 			"env":       containerInfo.environment,
+			"docker_host": common.MapStr{
+				"name": containerInfo.hostName,
+			},
```

Comment:
:+1: on renaming to 'hostname', let's keep `docker_host` until we come up with something better